### PR TITLE
Added french translation

### PR DESCRIPTION
**Pull request by TeddyCr**
### Describe your changes :
Added french translation.

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

Screenshot 2022-12-09 at 2 58 56 PM
Screenshot 2022-12-09 at 2 56 53 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by TeddyCr** [comment]
**Note** on the entity page, the buttons to add onwer and tier disapear when I view the french page. Do you know what could cause that?

File: openmetadata-ui/src/main/resources/ui/src/utils/i18next/i18nextUtil.ts [hunk 1]
```diff
@@ -13,15 +13,17 @@
...
 export const getInitOptions = (): InitOptions => {
   return {
-    supportedLngs: ['en-US'],
+    supportedLngs: ['en-US', 'fr-FR'],
     resources: {
       'en-US': { translation: enUS },
+      'fr-FR': { translation: frFR },
     },
-    fallbackLng: ['en-US'],
+    fallbackLng: ['fr-FR'],
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/i18next/i18nextUtil.ts:26]:
```suggestion
    fallbackLng: ['en-US'],
```

**Comment by Sachin-chaurasiya** [comment]
> **Note** on the entity page, the buttons to add onwer and tier disapear when I view the french page. Do you know what could cause that?

@TeddyCr , Can you share the screenshots or recording?, it would help for easy debugging.

File: openmetadata-ui/src/main/resources/ui/src/utils/i18next/i18nextUtil.ts (hunk 1 above)
Comment (TeddyCr) [reply openmetadata-ui/src/main/resources/ui/src/utils/i18next/i18nextUtil.ts]: Thanks, switched it to french for testing and forgot to switch it back.

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM, Thanks @TeddyCr.

**Comment by TeddyCr** [comment]
@Sachin-chaurasiya let me know if this helps 🙂


https://user-images.githubusercontent.com/13626425/206722404-06f59d2d-36cd-4a3a-b64f-805d9369e70d.mov

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya let me know if this helps 🙂
> 
>  Screen.Recording.2022-12-09.at.3.17.59.PM.mov

Thanks, @TeddyCr , @open-metadata/ui will have a look at this issue.

**Comment by Sachin-chaurasiya** [comment]
@TeddyCr , Can you please create a separate ticket for that?

**Pull request merged**